sdk/storage/Azure.Storage.Blobs/src/BlobExtensions.cs

```diff
@@ -142,5 +144,39 @@ internal static string ToTagsString(this Tags tags)
                 ExpiresOn = properties.ExpiresOn,
                 IsSealed = properties.IsSealed,
             };
+
+        /// <summary>
+        /// Internal. Parses Object Replication Policy ID from Rule ID and sets the Policy ID.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
If we decide to add docs to internal stuff then let's fill it properly. Otherwise we could remove it. Anyway it doesn't seem to bring more insight than signature does.

Reply:
Filled out the proper docs